.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: python
+python:
+  - "2.7"
+install:
+    - pip install yapf==0.6.2
+script:
+    - yapf --diff --exclude=./server/mlabns/third_party/* --recursive --style=google ./server/mlabns/*
```

Comment:
Can we add an equivalent pre-commit.sh as well (like in [telescope](https://github.com/m-lab/telescope/blob/development/dev/pre-commit.sh))?